src/pages/Home/component/Post/Post.tsx

```diff
@@ -13,7 +13,7 @@ const Post = (props: Props) => {
 	return (
 		<div className="post" key={post.id} onClick={() => onClick(post.id)}>
 			<p className="post__id">id: {post.id}</p>
-			<p className="post__title">title: {post.body}</p>
```

Comment:
So we do not want to render post's body?

Reply:
Yes, post body is for PostDetail